pkg/cmd/admin/registry/registry.go

```diff
@@ -190,6 +191,7 @@ func NewCmdRegistry(f *clientcmd.Factory, parentName, name string, out, errout i
 	cmd.Flags().StringSliceVar(&cfg.SupplementalGroups, "supplemental-groups", cfg.SupplementalGroups, "Specify supplemental groups which is an array of ID's that grants group access to registry shared storage")
 	cmd.Flags().StringVar(&cfg.FSGroup, "fs-group", "", "Specify fsGroup which is an ID that grants group access to registry block storage")
 	cmd.Flags().BoolVar(&cfg.DaemonSet, "daemonset", cfg.DaemonSet, "If true, use a daemonset instead of a deployment config.")
+	cmd.Flags().BoolVar(&cfg.Deployment, "deployment", cfg.Deployment, "If true, use a Kubernetes deployment instead of a deployment config.")
```

Comment:
@kargakis actually I think maybe we just need a template for registry instead of doing this.